src/main/java/org/apache/sling/auth/oauth_client/ClientConnection.java

```diff
@@ -34,4 +36,10 @@ public interface ClientConnection {
      * @return the name of the connection
      */
     @NotNull String name();
+    @NotNull String clientId();
```

Comment:
Same comment as in https://github.com/apache/sling-org-apache-sling-auth-oauth-client/pull/17#discussion_r2099767014 , please don't expose this kind of information in `ClientConnection`